client/components/student/student.service.js

```diff
@@ -7,8 +7,7 @@ app.factory('Student', function($resource) {
     id: '@_id'
   }, {
     list: {
-      method: 'GET',
-      isArray: true
+      method: 'GET'
```

Comment:
I made a bunch of changes throughout our app. You probably want to review and sync up before you make any more changes.